lib/beaker/dsl/install_utils.rb

```diff
@@ -453,12 +453,16 @@ def deploy_frictionless_to_master(host)
       # @api private
       #
       def do_install hosts, opts = {}
```

Comment:
@anodelman `do_install` actually doesn't take the masterless option, it overwrites it from the main options hash.  This happens a few lines below this comment.